docs/source/apps/projinfo.rst

```diff
@@ -0,0 +1,312 @@
+                LENGTHUNIT["metre",1]]],
+        PRIMEM["Greenwich",0,
+            ANGLEUNIT["degree",0.0174532925199433]],
+        CS[ellipsoidal,2],
+            AXIS["geodetic latitude (Lat)",north,
+                ORDER[1],
+                ANGLEUNIT["degree",0.0174532925199433]],
+            AXIS["geodetic longitude (Lon)",east,
+                ORDER[2],
+                ANGLEUNIT["degree",0.0174532925199433]],
+        USAGE[
+            SCOPE["unknown"],
+            AREA["World"],
+            BBOX[-90,-180,90,180]],
+        ID["EPSG",4326]]
+
+
+2. List the coordinate operations between NAD27 (designed with its CRS name)
+   and NAD83 (designed with its EPSG code 4269) within an area of interest
+
+.. code-block:: console
+
+        projinfo -s NAD27 -t EPSG:4269 --area "USA - Missouri"
+
+Output:
+
+::
+
+    DERIVED_FROM(EPSG):1241, NAD27 to NAD83 (1), 0.15 m, USA - CONUS including EEZ
+
+    PROJ string:
+    +proj=pipeline +step +proj=axisswap +order=2,1 +step +proj=unitconvert \
+    +xy_in=deg +xy_out=rad +step +proj=hgridshift +grids=conus \
+    +step +proj=unitconvert +xy_in=rad +xy_out=deg +step +proj=axisswap +order=2,1
+
+    WKT2_2018 string:
+    COORDINATEOPERATION["NAD27 to NAD83 (1)",
+        SOURCECRS[
+            GEOGCRS["NAD27",
+                DATUM["North American Datum 1927",
+                    ELLIPSOID["Clarke 1866",6378206.4,294.978698213898,
+                        LENGTHUNIT["metre",1]]],
+                PRIMEM["Greenwich",0,
+                    ANGLEUNIT["degree",0.0174532925199433]],
+                CS[ellipsoidal,2],
+                    AXIS["geodetic latitude (Lat)",north,
+                        ORDER[1],
+                        ANGLEUNIT["degree",0.0174532925199433]],
+                    AXIS["geodetic longitude (Lon)",east,
+                        ORDER[2],
+                        ANGLEUNIT["degree",0.0174532925199433]]]],
+        TARGETCRS[
+            GEOGCRS["NAD83",
+                DATUM["North American Datum 1983",
+                    ELLIPSOID["GRS 1980",6378137,298.257222101,
+                        LENGTHUNIT["metre",1]]],
+                PRIMEM["Greenwich",0,
+                    ANGLEUNIT["degree",0.0174532925199433]],
+                CS[ellipsoidal,2],
+                    AXIS["geodetic latitude (Lat)",north,
+                        ORDER[1],
+                        ANGLEUNIT["degree",0.0174532925199433]],
+                    AXIS["geodetic longitude (Lon)",east,
+                        ORDER[2],
+                        ANGLEUNIT["degree",0.0174532925199433]]]],
+        METHOD["CTABLE2"],
+        PARAMETERFILE["Latitude and longitude difference file","conus"],
+        OPERATIONACCURACY[0.15],
+        USAGE[
+            SCOPE["unknown"],
+            AREA["USA - CONUS including EEZ"],
+            BBOX[23.81,-129.17,49.38,-65.69]],
+        ID["DERIVED_FROM(EPSG)",1241]]
+
+Since
+*****
+
+.. versionadded:: 6.0.0
```

Comment:
I have generally tried to keep the `versionadded` tag at the top of the section that is describing the thing that was added. If that pattern is followed this should go to the top of the document, right under the `projinfo` header. See for example https://github.com/OSGeo/proj.4/blob/master/docs/source/operations/projections/bertin1953.rst

Probably best to include it in the `.. only:: html` section to avoid cluttering the man page.